convert2rhel/unit_tests/toolopts_test.py

```diff
@@ -68,3 +71,17 @@ def test_cmdline_disablerepo_defaults_to_asterisk(self):
     @unit_tests.mock(sys, "argv", _params(["--disable-submgr"]))
     def test_cmdline_exits_on_empty_enablerepo_with_disable_submgr(self):
         self.assertRaises(SystemExit, convert2rhel.toolopts.CLI)
+
+
+@pytest.mark.parametrize(
+    ("argv"), [
+        _params(["-v", "Server"]),
+        _params(["--variant", "Client"]),
+        _params(["--variant"])
+    ]
```

Comment:
You're testing only when if the condition is True, but also it should be tested when it False. In this case you can check that ask_to_continue wasn't called, or log records doesn't contains something ...